### ISSUE TITLE
Burnout Paradise: The Ultimate Box (24740)

### ISSUE THREAD

**Issue by diegogangl** [ValveSoftware/Proton#468]
Game runs perfect with no performance issues at all. It didn't recognize my cheap usb gamepad but TBH I don't remember if the windows version did either.

- Ubuntu 16.04 (KDE Neon)
- GTX 980
- Nvidia drivers 396.51
- Kernel 4.15.0

**Comment by hamid-elaosta** [comment]
Switching to dual or triple monitor in-game doesn't work and squashes all 2 or 3 screens into the one screen.

I tested on Ubuntu 18.04.1 (Gnome3), GTX 1080 Ti O11G, NVIDIA 396.54, Kernel 4.15, other than multi monitor, seems to work fine, performance is fine.

**Comment by LaniusFNV** [comment]
The multi monitor setup requires you to define one big display compromising however many monitors you wish to use. This would be done using TwinView (using Nvidia) or Xinerama under Linux. [Arch Wiki] 
This behaviour is used on Windows as well and isn't an error.

[Arch Wiki]: https://wiki.archlinux.org/index.php/Multihead#TwinView

**Comment by thunder1410** [comment]
How do you start the game? It tells me that I need to reinstall origin and I don't know if or how I can do that through proton

**Comment by kisak-valve** [comment]
Burnout paradise can't start on resume

Issue transferred from https://github.com/ValveSoftware/Proton/issues/729.
@SaulNunez posted on 2018-08-27T02:25:24:

On first start Burnout paradise runs ridiculously good. However after closing, probably after the save creation we can't return to that save file. It keeps on the loading screen previous to the car selection.

**Comment by thunder1410** [comment]
I had to verify the game cache to fix my issue. Performance isn't great around 40fps on medium full HD.
system information: https://gist.github.com/thunder1410/76897c9bd211c74b30968baa4037ce96
EDIT: can confirm I can restart the game and continue from the save file

**Comment by Dazarlin** [comment]
I can confirm that the game runs really well on medium settings and can in fact resume a previous save on restart.
You do however need to go into the control options and set up the controls for a controller manually it seems. Ran pretty much flawlessly for me. Could easily see this being a whitelisted game from my experience.
System Info: https://gist.github.com/Dazarlin/1fe5b3d929f71f1f3f598c5b99d0c262

EDIT: In my experience the performance was the same even after restarting the game.

**Comment by parkerlreed** [comment]
It doesn't see any controllers as far as I can see

![image](https://user-images.githubusercontent.com/841440/44716117-72e95000-aa87-11e8-8320-7bb233d1b5ca.png)

Have my Steam controller hooked up setup as plain gamepad (360)

EDIT: System info https://gist.github.com/parkerlreed/bca1635059e2ac56ea35e5e35ea443fd

Tried both runtime and native. On the Proton 3.7-4 beta

**Comment by parkerlreed** [comment]
Log
[steam-24740.log](https://github.com/ValveSoftware/Proton/files/2327582/steam-24740.log)

**Comment by hamid-elaosta** [comment]
@parkerlreed I also don't see my controller. Xbox One S controller connected via Bluetooth. 
Can post logs when I'm home.

**Comment by Dazarlin** [comment]
I got the same screen as @parkerlreed at first. The way I got the controller to work for me was:

1) Have the controller plugged in before starting the game.
2) Click on one of those options to rebind it and then press the input on the controller you want it to be mapped to.

That somehow magically worked for me. At that point it recognized my controller. It only showed my controller in game after I'd mapped something to the controller. Hope this helps.

**Comment by parkerlreed** [comment]
@Dazarlin Thanks. I'll give that a shot a little later.

**Comment by hamid-elaosta** [comment]
@Dazarlin @parkerlreed Can confirm, tapping on a button to map it, pops up my Xbox One S controller (Bluetooth) and allows me to map it. Sadly there is no default mapping so you have to map each key manually by selecting it with the keyboard and then pressing the mapping you want on the controller.

**Comment by parkerlreed** [comment]
I could have sworn it supported X360 out of the box on Windows. At least this works (judging by the two reports)

Thanks for testing!

**Comment by Dazarlin** [comment]
I use a Xbox 360 controller and used the exact same one back when I used Windows and can confirm that it did work out of the box on there. It still works on Linux through Proton, but as stated above you need to set the controls manually for it to work. It's something at least.

**Comment by parkerlreed** [comment]
Confirmed here. Steam controller acting as normal gamepad maps fine.

**Comment by adolson** [comment]
Works exceptionally well on my machine (i5 4690, GTX 660OC 2GB, 24GB DDR3, DualShock 4, Ubuntu 18.10, Steam beta branch, Proton 3.16-4). Had to configure the gamepad like everyone else, but it works fine. I had to drop AA from 16x to 4x for it to look smooth, and had to log in to support.ea.com and register the product key to get online to work. The ONLY issue I see is that if I choose 3rd person camera while driving a bike, the rider's head is missing. lol

**Comment by bradgy** [comment]
Performance on my machine is outstanding, however I get crashes to desktop at seemingly random times - eg. after races, and on pressing start to get to the menu.
[steam-24740.log](https://github.com/ValveSoftware/Proton/files/4469503/steam-24740.log)
Happens w Proton 5.05 and next -.
i3 8100 RX580 Mesa 20.0.4 16GB Ubuntu 20.04 Steam Controller Steam Beta Client
Edit: update, crashes only seem to happen when game launched from BPM.

**Comment by bradgy** [comment]
> Performance on my machine is outstanding, however I get crashes to desktop at seemingly random times - eg. after races, and on pressing start to get to the menu.
> [steam-24740.log](https://github.com/ValveSoftware/Proton/files/4469503/steam-24740.log)
> Happens w Proton 5.05 and next -.
> i3 8100 RX580 Mesa 20.0.4 16GB Ubuntu 20.04 Steam Controller Steam Beta Client
> Edit: update, crashes only seem to happen when game launched from BPM.

Actually, just got a crash in regular client, not in BPM. 
[steam-24740.log](https://github.com/ValveSoftware/Proton/files/4483715/steam-24740.log)

**Comment by AsciiWolf** [comment]
Not Proton related, but was anyone able to workaround the login issue? Servers for the original Burnout Paradise do not seem to be online anymore and login dialog appears every time the game is started. It is possible to enter random username/password to get into game, however it would be great if this dialog could be skipped.

**Comment by parkerlreed** [comment]
That dialog should let you log into Origin, it did at least the last time I tried. 

Did they nuke the login server as well after Remastered came out?

**Comment by AsciiWolf** [comment]
Yeah, it unfortunately looks like that the login server got nuked as well. :-/

**Comment by parkerlreed** [comment]
With my existing login the only thing I get is a "Hey the servers are down press ok"

I'd share the AutoLogin file but for obvious reasons... :( That sucks.

**Comment by parkerlreed** [comment]
Also the other bad news. I bought Remastered just to try out. It uses about the same Save location (Just the name of the game changed) but the Profile.BurnoutParadiseSave isn't compatible

It detects the older save as corrupted and just creates a new one. 

Plus the new save file format is about twice as big, making conversion a pain

```
-rw-r--r-- 1 parker parker 629K Apr 21 13:27  Profile.BurnoutParadiseSave
-rw-r--r-- 1 parker parker 373K Apr 21 13:27  Profile.BurnoutParadiseSave_old
```

I've been trying to dig into it but haven't come up with anything yet. I didn't want to completely restart.

**Comment by vadiof** [comment]
SInce Proton 5.0.8 game launch is broken somehow. 4.11-13 works perfect though

**Comment by kisak-valve** [comment]
Hello @vadiof, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the issue with Proton 5.0 and drag and drop the generated $HOME/steam-$APPID.log into the comment box. Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by vadiof** [comment]
@kisak-valve all done!
https://gist.github.com/vadiof/38cb5ef6e21f1369812adaae63336831
[steam-24740.log](https://github.com/ValveSoftware/Proton/files/4776751/steam-24740.log)

**Comment by kisak-valve** [comment]
Thanks, your system information tells us that you do not have 32 bit vulkan drivers installed on your system. Please run `sudo apt install mesa-vulkan-drivers:i386`.

**Comment by K0bin** [comment]
[burnout_bt_all.txt](https://github.com/ValveSoftware/Proton/files/10052043/burnout_bt_all.txt)

Backtrace of all threads of the hang before the menu.

**Comment by Leopard1907** [comment]
This game fails to boot with high thread count cpus, my former system that had a 4C/8T cpu was able to run the game without any workarounds, my 8C/16T system needs this as a launch command to work:

`WINE_CPU_TOPOLOGY=4:0,1,2,3 %command%`

**Comment by ghost** [comment]
If the game is installed with Steam Play, DirectX is not automatically installed which causes an issue for the game.

Solution: Install DirectX, which is provided in the game files.

It is recommended for Valve to install the provided DirectX that comes with the game files as a part of a script when installing the game with Steam Play.
